data/World/Overworld.json

```diff
@@ -1385,8 +1385,31 @@
     {
         "region_name": "Market Treasure Chest Game",
         "scene": "Market Treasure Chest Game",
+        "hint": "MARKET",
```

Comment:
This changes the treasure chest game to be considered part of the market hint area regardless of where it's shuffled to. I think this is unintuitive due to its inconsistency with other interiors and should be reverted.

Reply:
This was to troubleshoot a seed generation error due to hint region.   We fixed it and forgot to remove it, we'll remove it

Reply:
addressed